Modules/rat_rescue.py

```diff
@@ -150,25 +144,10 @@ def __eq__(self, other) -> bool:
     def __hash__(self):
 
         if self._hash is None:
-            attributes = (
-                self.uuid,
-                self.board_index,
-                self.client,
-                self.platform,
-                self.first_limpet,
-                self.created_at,
-                self.updated_at,
-                self.system,
-                self.active,
-                self.code_red,
-                self.outcome,
-                self.title,
-                self.first_limpet,
-                self.lang_id,
-                self.irc_nickname,
-            )
-
-            self._hash = reduce(xor, map(hash, attributes))
+            # compute the hash of the internally generated uuid and use that as our hash
+            # this resolves SPARK-52
```

Comment:
I don't believe source code comments should be a place to mention which issue is resolved by which piece of code. We have PRs and commit messages for that.